packages/keystore/src/universal-crypto.ts

```diff
@@ -0,0 +1,34 @@
+import type { createCipheriv, createDecipheriv } from 'crypto';
```

Comment:
This is working on a web app mainly with typescript also?

Reply:
This is a nice question, I thought the same thing when I was asking haha...

We should open an issue to have `SDK tests on browser compatibility` in the future.

Reply:
https://github.com/FuelLabs/fuels-ts/issues/284